addons/website_sale/models/website.py

```diff
@@ -1007,3 +1007,45 @@ def _get_canonical_url(self):
 
     def _get_snippet_defaults(self, snippet):
         return super()._get_snippet_defaults(snippet) | const.SNIPPET_DEFAULTS.get(snippet, {})
+
+    def _get_product_image_ratio(self):
+        """Get the product image aspect ratio based on the website's design classes.
+
+        Returns:
+            str: The aspect ratio as a string (e.g., '16/9', '4/3', '1/1')
+        """
+        classes = self.shop_opt_products_design_classes or ''
+        ratio_mapping = {
+            'o_wsale_products_opt_thumb_16_9': '16/9',
+            'o_wsale_products_opt_thumb_4_3': '4/3',
+            'o_wsale_products_opt_thumb_6_5': '6/5',
+            'o_wsale_products_opt_thumb_4_5': '4/5',
+            'o_wsale_products_opt_thumb_2_3': '2/3',
```

Comment:
For technical reasons, it would be easier to keep underscore there. See my next comment.
```suggestion
            'o_wsale_products_opt_thumb_16_9': '16_9',
            'o_wsale_products_opt_thumb_4_3': '4_3',
            'o_wsale_products_opt_thumb_6_5': '6_5',
            'o_wsale_products_opt_thumb_4_5': '4_5',
            'o_wsale_products_opt_thumb_2_3': '2_3',
```